plenum/server/consensus/consensus_shared_data.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, name: str, validators: List[str], inst_id: int):
         self._low_watermark = 0
         self.log_size = 300  # TODO: use config value
         self._high_watermark = self._low_watermark + self.log_size
+        self._total_nodes = 0
```

Comment:
I believe we already can get the total number of nodes: either as `len(self.validators)` or `self.quorums.n`.